src/main/java/frc/robot/subsystems/pivot/PivotIOSparkMax.java

```diff
@@ -0,0 +1,87 @@
+package frc.robot.subsystems.pivot;
+
+import static frc.robot.Constants.PivotConstants.*;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.trajectory.ExponentialProfile;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+
+public class PivotIOSparkMax implements PivotIO {
+  private boolean openLoop = false;
+  private double appliedVolts = 0.0;
+  private final CANSparkMax leftMotor =
+      new CANSparkMax(LEFT_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+  private final CANSparkMax rightMotor =
+      new CANSparkMax(RIGHT_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+  private final DutyCycleEncoder absoluteEncoder = new DutyCycleEncoder(ENCODER_PORT);
+  private final PIDController controller = new PIDController(50, 0.0, 0.0);
```

Comment:
It would help new people if you named it feedback controller or something like this